src/main/java/seedu/address/logic/commands/DownloadAbstract.java

```diff
@@ -34,24 +34,28 @@
 
     protected static final String IVLE_LOGIN_BUTTON_ID = "ctl00_ctl00_ContentPlaceHolder1_btnSignIn";
 
-    protected static final String WRONG_PASS_USER_MESSAGE = "WRONG PASSWORD OR USERNAME ENTERED";
+    protected static final String MESSAGE_USERNAME_PASSWORD_ERROR = "WRONG PASSWORD OR USERNAME ENTERED";
```

Comment:
The error message should not be in caps lock. Please take reference from existing error messages. They are typically "Error name!", followed by a correct example of how to use the command.